### PR TITLE
Added scripts to make PR2_v5 and a combined EUK_PRO database from PR2v5 and Silva138

### PR DESCRIPTION
**Pull request by MDHDZ91**
the PR2_v5 steps were created following the scripts for the v4 version with minimal changes.
the EUK_PRO database scripts were created using guidance from Jesse

### CODE REVIEW

**Comment by jcmcnch** [comment]
Hi María, 

This all looks great! Thanks for putting this together - I'm sure it will be useful to @shanexuuu and others who want to remake the bbsplit db. One minor change - for the shebang line, can you please change it to `#!/bin/bash -i`, the `-i` flag ensures the scripts behave the same way as they would in an interactive terminal which is a safeguard against unexpected behaviour. Once you change that I'd be happy to merge.

Cheers,
Jesse

**Comment by MDHDZ91** [comment]
okay I added the -i flag, thank you for that suggestion!

**Review by jcmcnch — APPROVED** [review]

File: PR2_v5/00-prep-raw_PR2_files/script_to_reformat_PR.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#this script will extract the headers from the .fasta and create a table with feature/taxonomy
+
+##download the file
+wget https://github.com/pr2database/pr2database/releases/download/v5.0.0/pr2_version_5.0.0_SSU_dada2.fasta.gz
+## unzip
+gzip -d pr2_version5.0.0_SSU_dada2.fasta.gz
+
+
+# Input file
+input_file="pr2_version5.0.0_SSU_dada2.fasta"
```
Comment (nahlgren) [inline PR2_v5/00-prep-raw_PR2_files/script_to_reformat_PR.sh:12]: I think there's a small error in the file names on lines 8 and 12
Should be pr2_version_5.0.0_SSU_dada2.fasta and pr2_version_5.0.0_SSU_dada2.fasta.gz, if I'm not mistaken

Comment (jcmcnch) [reply PR2_v5/00-prep-raw_PR2_files/script_to_reformat_PR.sh:12]: Hi Nathan - not sure, I didn't do this bit (this is Maria's contribution). If you're confident this is a mistake though, you could just open a pull request and I can merge it.

**Comment by MDHDZ91** [comment]
line 8 specifically uses gzip so you need the .gz version there, and
line 8 uses the unzipped file



María D. Hernández Limón
Coleman Lab  <http://biogeolabs.uchicago.edu/colemanlab/>
PhD Candidate
Dept. of the Geophysical Sciences
University of Chicago
(224)-772-4017

UChicago-DAB Member
<https://voices.uchicago.edu/uchicagograd/2019/10/17/meet-the-uchicagograd-diversity-advisory-board-dab/>
| PSD EDI Student Committee Member
<https://physicalsciences.uchicago.edu/about/diversity-inclusion/committees/edi-student-committee/>
Neubauer Fellow <https://grad.uchicago.edu/fellowships/neubauer-fellows/>|DSEER
NSF Fellow <https://voices.uchicago.edu/nrtenergyandenvironment/about/>| Ford
Foundation Predoctoral Fellow
<https://sites.nationalacademies.org/PGA/FordFellowships/index.htm>

**Pull request merged**